libraries/AP_Beacon/AP_Beacon.h

```diff
@@ -116,4 +123,9 @@ class AP_Beacon
     // individual beacon data
     uint8_t num_beacons = 0;
     BeaconState beacon_state[AP_BEACON_MAX_BEACONS];
+
+    // fence boundary
+    Vector2f *_boundary;
```

Comment:
Can you remove the underscore? Either we have them or not, mixing the two inside the same class is bad though.